tensorflow_datasets/scripts/replace_fake_images.py

```diff
@@ -82,7 +82,7 @@ def rewrite_zip(root_dir, zip_filepath):
     rewrite_dir(temp_dir)  # Recursivelly compress the archive content
 
     # Compressed the .zip file again
-    with zipfile.ZipFile(zip_filepath, 'w') as zip_file:
+    with zipfile.ZipFile(zip_filepath, 'w',compression=zipfile.ZIP_DEFLATED) as zip_file:
```

Comment:
Can you add a comment explaining how the compression method was choose ? `compression=zipfile.ZIP_DEFLATED`